modules/tinymce/src/core/main/ts/api/Settings.ts

```diff
@@ -118,6 +118,12 @@ const getContentCss = (editor: Editor): string[] => {
   }
 };
 
+const getFontCss = (editor: Editor): string[] => {
+  const fontCss = editor.getParam('fonts_css', []);
```

Comment:
This should probably be `font_css` rather than `fonts_css` so as it's consistent with `content_css` as it's not `contents_css`.